.meta/dietpi-imager

```diff
@@ -115,19 +115,22 @@
 		OUTPUT_IMG_NAME=$G_WHIP_RETURNED_VALUE
 
 		# Check for existing file, in case offer backup. Skip if source is image file and matches output file already.
-		if [[ /root/$OUTPUT_IMG_NAME.img != $FP_SOURCE_IMG && -f /root/$OUTPUT_IMG_NAME.img ]]; then
+		if [[ $OUTPUT_IMG_NAME.img != $FP_SOURCE_IMG && -f $OUTPUT_IMG_NAME.img ]]; then
 
 			G_WHIP_BUTTON_OK_TEXT='Overwrite'
 			G_WHIP_BUTTON_CANCEL_TEXT='Backup'
 			G_WHIP_YESNO "[WARNING] /root/$OUTPUT_IMG_NAME.img already exists\n
-Do you want to overwrite or backup the existing file to $OUTPUT_IMG_NAME.img.bak?" || mv /root/$OUTPUT_IMG_NAME.img /root/$OUTPUT_IMG_NAME.img.bak
+Do you want to overwrite or backup the existing file to /root/$OUTPUT_IMG_NAME.img.bak?" || mv $OUTPUT_IMG_NAME.img{,.bak}
```

Comment:
`mv $OUTPUT_IMG_NAME.img{,.bak}` nice :+1:, never through about using this for creating backup files, will keep it in mind for other cases as well.